src/lib/blkalloc/append_blk_allocator.h

```diff
@@ -137,12 +118,12 @@ class AppendBlkAllocator : public BlkAllocator {
     void on_meta_blk_found(const sisl::byte_view& buf, void* meta_cookie);
 
 private:
-    std::mutex m_mtx;                  // thread_safe, TODO: open option for consumer to choose to go lockless;
-    blk_num_t m_last_append_offset{0}; // last appended offset in blocks;
-    blk_num_t m_freeable_nblks{0};
+    std::atomic< blk_num_t > m_last_append_offset{0}; // last appended offset in blocks;
```

Comment:
Maybe put some comment say, 
`m_last_append_offset` is in memory only.
`m_commit_offset` is the persistence version (going to be persisted) in next cp.

Reply:
Done